src/imars3dv2/filters/rotation.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import numpy as np
+import tomopy
+import concurrent.futures as cf
+import tomopy.util.mproc as mproc
+from tomopy.recon.rotation import find_center_pc
+from imars3dv2.filters.tilt import find_180_deg_pairs_idx
```

Comment:
Since finding 180 pairs is "common" would it be good to move it into a different place? `rotation` module depending on `tilt` module seems odd. Off the top of my head, we could export it in `imars3dv2.filters` itself.

Reply:
`find_180_deg_pairs_idx` is now available at the module `filters` level.